### PR TITLE
[Behat] Retag behat features

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Fixed tickets | #4149

### CODE REVIEW

File: features/shop/round_taxes_on_order_item_level.feature [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@ui-cart
+@ui @cart
```
Comment (pjedrzejewski) [inline features/shop/round_taxes_on_order_item_level.feature]: I think it should be the following order ``@cart @domain @api @ui``. Thoughts?

Comment (pamil) [inline features/shop/round_taxes_on_order_item_level.feature]: `@cart` should be the first one, I'm not sure about the other ones order.

Comment (michalmarcinkowski) [inline features/shop/round_taxes_on_order_item_level.feature]: :+1:

Comment (lchrusciel) [inline features/shop/round_taxes_on_order_item_level.feature]: We will implement *ui* before *api* probably, so I would vote to swap them. Rest is :+1:

Comment (pjedrzejewski) [inline features/shop/round_taxes_on_order_item_level.feature]: Yeah, let's do this.

Comment (lchrusciel) [inline features/shop/round_taxes_on_order_item_level.feature]: :+1: done

File: features/legacy/backend/metadata/metadata_customization.feature [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@metadata @ui
+@legacy_metadata @ui
```
Comment (michalmarcinkowski) [inline features/legacy/backend/metadata/metadata_customization.feature]: Why do we need `ui` tag here? /cc @pamil

Comment (pamil) [inline features/legacy/backend/metadata/metadata_customization.feature]: It's not needed, can be deleted.

Comment (lchrusciel) [inline features/legacy/backend/metadata/metadata_customization.feature]: :+1:

**Pull request merged**